Composer/packages/electron-server/src/preload.js

```diff
@@ -3,6 +3,8 @@
 
 const { ipcRenderer, contextBridge } = require('electron'); // eslint-disable-line
 
+const isOneAuthEanabled = !(process.env.COMPOSER_ENABLE_ONEAUTH === 'false' || process.platform === 'linux');
```

Comment:
Note we can't extract this into the module and share it with `Composer/packages/electron-server/src/auth/oneAuthService.ts` due to the preloader.js being used as is without bundling.